src/interface/report/Results/Character/PlayerInfo/PlayerGearHeader.tsx

```diff
@@ -1,10 +1,15 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
 
 import { makeCharacterUrl } from 'interface/common/makeAnalyzerUrl';
+import Combatant from 'parser/core/Combatant';
 
-const PlayerGearHeader = ({ player, averageIlvl }) => (
+interface Props {
+  player: Combatant;
+  averageIlvl: number;
+}
+
+const PlayerGearHeader = ({ player, averageIlvl }: Props) => (
   <div className="player-gear-header">
     <div className={`${player.spec.className.replace(' ', '')} player-name`}>
       <Link to={makeCharacterUrl(player)}>{player.name}{player.characterProfile && <> - {player.characterProfile.realm}</>}</Link>
```

Comment:
Centering this already made it prettier, but I think moving the complete realm name to the second line would look a bit cleaner compared to `Synida - Laughing \n Skull`.

Talking about something like this `{player.characterProfile && <><br />{player.characterProfile.realm}</>}`, or maybe check the length of character & realm to see if it fits in a single line or should break between char and realm (since `.player-gear-header` is always roughly the same width)

Reply:
Great suggestion, this is the new look:
<img width="638" alt="Screenshot 2020-10-29 at 23 41 22" src="https://user-images.githubusercontent.com/3826157/97640747-b9f57600-1a41-11eb-866a-11bf92ceddda.png">